ember/testem.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   test_page: "tests/index.html?hidepassed",
   disable_watching: true,
   launch_in_ci: ["Chrome"],
-  launch_in_dev: ["Chrome"],
+  launch_in_dev: [],
```

Comment:
Out of curiosity: why?

Reply:
Otherwise, Test'em keeps opening new Chrome windows. I like to choose myself which browser I use and when I open a new window/tab. Automatically launching a browser is IMO only useful when running it headless.

Reply:
Totally agree, thanks!